samples/RTOS/Basic-Test/main.cpp

```diff
@@ -0,0 +1,285 @@
+//
```

Comment:
Delete this automatically generated comment and add a comment that follows our normal standard